malcolm/modules/pmac/parts/cspart.py

```diff
@@ -28,6 +29,20 @@ def setup(self, registrar):
         # Add methods
         registrar.add_method_model(
             self.move, "moveCS%d" % self.cs, needs_context=True)
+        # Hooks
+        registrar.hook(scanning.hooks.ReportStatusHook, self.report_status)
+
+    @add_call_types
+    def report_status(self, context):
```

Comment:
I don't think anything actually consumes this info, so you can delete all the changes in this file